stl/inc/optional

```diff
@@ -394,7 +394,7 @@ public:
     }
 
     template <class _Ty2>
-    _NODISCARD constexpr _Ty value_or(_Ty2&& _Right) const& {
+    _NODISCARD constexpr remove_cv_t<_Ty> value_or(_Ty2&& _Right) const& {

```

Comment:
I've submitted an LWG issue (but not yet been assigned a number) to make the changes to both `value_or` overloads Standard.